cfpb/consumerfinance.gov
# Only update the iRegs wayfinder if there's something to update it with

@anselmbradford This should hopefully catch the errors we're seeing in NR.

## Testing

1. Notice the console errors at http://localhost:8000/rules-policy/regulations/1003/4/#blah
1. Pull and build this branch.
1. No more console errors.
